ks3-standard/src/commonTest/kotlin/io/ks3/standard/EnumAsOrdinalSerializerTest.kt

```diff
@@ -0,0 +1,52 @@
+package io.ks3.standard
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+object ShapeSerializer : KSerializer<Shape> by enumAsOrdinalSerializer()
+
+class EnumAsOrdinalSerializerTest : FreeSpec(
+   {
```

Comment:
What formatter are you using? It looks odd to see the brackets on separate lines. I'd prefer this:

```kotlin
class EnumAsOrdinalSerializerTest : FreeSpec({
   // ...
})
```

I *really* don't want to include anything like ktlint - I think committing a `.editorconfig` file or `.idea/codestyles` is fine.

Reply:
I personally used ktlint a lot, but I'm open for not using it. I'll disable it for now. If you have any codestyles you'd like to commit feel free to open a PR